boot.php

```diff
@@ -92,21 +72,7 @@ function local_user()
  */
```

Comment:
Please mark this method as deprecated
```suggestion
 * @deprecated since version 2022.12, use Core\Session:: getPublicContact() instead
 */
```